app/controllers/claims/schools_controller.rb

```diff
@@ -1,5 +1,15 @@
 class Claims::SchoolsController < ApplicationController
+  before_action :has_one_school?, only: :index
```

Comment:
Maybe a nitpick, but can we rename this to show what it does?

```suggestion
  before_action :redirect_to_school_when_belongs_to_one_school, only: :index
```